src/plugins/synchronizedFolder/SynchronizedObjectsProvider.js

```diff
@@ -0,0 +1,75 @@
+export const PLANS_FOLDER_KEY = 'folder_synchronized_couchdb';
+const COUCHDB_PERSISTENCE_SPACE = 'mct';
+const MY_ITEMS_LOCATION = 'mine';
+
+export class SynchronizedObjectsProvider {
+    constructor(openmct, options) {
+        this.openmct = openmct;
+        this.filter = options.filter;
+        const identifier = {
+            namespace: COUCHDB_PERSISTENCE_SPACE,
+            key: PLANS_FOLDER_KEY
+        };
+
+        this.openmct.objects.get(identifier).then((domainObject) => {
```

Comment:
Probably need to have some check here to not add the folder if CouchDB is not being used.
I will think about how, but any suggestions are welcome.